### PR TITLE
New payment page followup

### PR DESCRIPTION
**Pull request by ds300**
Follow up from #2025 (addressing the feedback in the comments)

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit 3b59952088c28c3f1833407cbfca4e7bca8ac071

https://deploy-preview-2032--artsy-reaction.netlify.com

**Comment by sweir27** [comment]
🙌 🙌

File: src/Apps/Order/Routes/__tests__/NewPayment.test.tsx [hunk 1]
```diff
@@ -0,0 +1,528 @@
...
+
+import { trackPageView } from "Apps/Order/Utils/trackPageView"
+import { createTestEnv } from "DevTools/createTestEnv"
+import moment from "moment"
+import { graphql } from "react-relay"
+import { Address, AddressForm } from "../../Components/AddressForm"
+import { NewPaymentFragmentContainer } from "../NewPayment"
+import { OrderAppTestPage } from "./Utils/OrderAppTestPage"
+
+const fillAddressForm = (component: any, address: Address) => {
```
Comment (sweir27) [inline src/Apps/Order/Routes/__tests__/NewPayment.test.tsx]: When we look to refactor these payment pages, I think this method is repeated across a few test files.

**Comment by artsyit** [comment]
:rocket: PR was released in v13.0.8 :rocket:

**Pull request merged**